src/engine/strat_engine/filesystem.rs

```diff
@@ -147,10 +147,15 @@ pub fn mount_fs(dev_path: &Path, mount_point: &Path) -> EngineResult<()> {
     Ok(())
 }
 
-pub fn unmount_fs(mount_point: &Path) -> EngineResult<()> {
+pub fn unmount_fs(mount_point: &Path, flags: &str) -> EngineResult<()> {
```

Comment:
My first atttempt used:

 let output = try!(Command::new("umount").arg(mount_point).args(flags).output());

But, if the user passed in "" - the umount command failed.